core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java

```diff
@@ -1216,4 +1219,25 @@ private void applyLockTime(long lockTime, Transaction tx) {
         tx.getInputs().forEach(input -> input.setSequenceNumber(TransactionInput.NO_SEQUENCE - 1));
         tx.setLockTime(lockTime);
     }
+
+    private boolean removeDust(Transaction tx) {
+        List<TransactionOutput> txos = tx.getOutputs();
+        List<TransactionOutput> toKeep = new ArrayList<TransactionOutput>();
+        for (TransactionOutput zz: txos) {
```

Comment:
...especially names like `zz` are very hard to follow except in the most simple cases. Calling it `output` or `txOutput` to make it more readable is preferred.